x-pack/metricbeat/module/aws/elb/_meta/data.json

```diff
@@ -26,7 +58,6 @@
         "name": "elb"
     },
     "service": {
-        "name": "cloudwatch",
         "type": "cloudwatch"
```

Comment:
I wonder why `service.type` is still set to `cloudwatch` :thinking:

Reply:
Yeah I realized all aws metricsets have the wrong `service.type` so I left it and was thinking to create a separate PR to fix it. But I guess adding in this PR is fine.